Userland/Libraries/LibSoftGPU/Device.cpp

```diff
@@ -189,7 +191,7 @@ void Device::rasterize_triangle(Triangle const& triangle)
     GPU::Vertex const& vertex1 = triangle.vertices[1];
     GPU::Vertex const& vertex2 = triangle.vertices[2];
 
-    // Calculate area of the triangle for later tests
+    // Determine area of the triangle to calculate barycentrics
```

Comment:
Determine area of the ~~triangle~~ parallelogram to calculate barycentrics. By your own words ;)

Reply:
Done, thanks!